src/ui/package.json

```diff
@@ -10,35 +10,38 @@
     "test-ci": "ng test --single-run",
     "pree2e": "webdriver-manager update",
     "e2e": "protractor",
-    "compile": "ng build --prod --progress=false -o rootfs/dist",
-    "compile-aot": "ng build --prod --aot --progress=false -o rootfs/dist"
+    "compile": "ng build --prod --progress=false --output-path rootfs/dist",
+    "compile-aot": "ng build --prod --aot --progress=false --output-path rootfs/dist"
   },
   "private": true,
   "dependencies": {
-    "@angular/common": "2.4.3",
-    "@angular/compiler": "2.4.3",
-    "@angular/core": "2.4.3",
-    "@angular/forms": "2.4.3",
-    "@angular/http": "2.4.3",
+    "@angular/animations": "^4.0.0",
```

Comment:
Is there any break-change between angular version 2 and 4? I think this is a huge change. I found some break-changes between minor versions